packages/medusa/src/services/event-bus.ts

```diff
@@ -167,7 +200,7 @@ export default class EventBusService {
   async emit<T>(
     eventName: string,
     data: T,
-    options: EmitOptions = {}
+    options: EmitOptions = { attempts: 1 }
```

Comment:
self-review: Bull will set attempts to 1 if not provided, so I added it as a default option.